fe/fe-core/src/main/java/org/apache/doris/job/extensions/insert/InsertJob.java

```diff
@@ -75,55 +118,147 @@ public class InsertJob extends AbstractJob<InsertTask, Map> {
         COLUMN_TO_INDEX = builder.build();
     }
 
-    @SerializedName(value = "lp")
-    String labelPrefix;
+    @SerializedName("taskIdList")
+    ConcurrentLinkedQueue<Long> taskIdList;
 
-    InsertIntoTableCommand command;
+    private final long dbId;
+    private String labelName;
+    private List<InsertIntoTableCommand> plans;
+    private InsertJob.LoadType loadType;
+    // 0: the job status is pending
+    // n/100: n is the number of task which has been finished
+    // 99: all tasks have been finished
+    // 100: txn status is visible and load has been finished
+    private int progress;
+    private long createTimestamp = System.currentTimeMillis();
+    private FailMsg failMsg;
+    private LoadStatistic loadStatistic = new LoadStatistic();
+    private Set<Long> finishedTaskIds = new HashSet<>();
+    private Set<String> tableNames;
+    private ConcurrentHashMap<Long, InsertTask> idToTasks = new ConcurrentHashMap<>();
+    private Map<String, String> properties;
+    private AuthorizationInfo authorizationInfo;
+    private final ReentrantReadWriteLock lock = new ReentrantReadWriteLock(true);
+
+    private ConnectContext ctx;
+    private StmtExecutor stmtExecutor;
+    private List<ErrorTabletInfo> errorTabletInfos = new ArrayList<>();
+    private List<TabletCommitInfo> commitInfos = new ArrayList<>();
 
-    StmtExecutor stmtExecutor;
+    // max save task num, do we need to config it?
+    private static final int MAX_SAVE_TASK_NUM = 100;
 
-    ConnectContext ctx;
+    /**
+     * load job type
+     */
+    public enum LoadType {
+        BULK,
+        SPARK,
+        LOCAL_FILE,
+        UNKNOWN
 
-    @SerializedName("tis")
-    ConcurrentLinkedQueue<Long> taskIdList;
+    }
 
-    // max save task num, do we need to config it?
-    private static final int MAX_SAVE_TASK_NUM = 100;
+    public InsertJob(ReplayLoadLog.ReplayCreateLoadLog replayLoadLog) {
+        super(replayLoadLog.getId());
+        setJobId(replayLoadLog.getId());
+        this.dbId = replayLoadLog.getDbId();
+    }
+
+    public InsertJob(Long jobId, String jobName,
+                     JobStatus jobStatus,
+                     String currentDbName,
+                     String comment,
+                     UserIdentity createUser,
+                     JobExecutionConfiguration jobConfig,
+                     Long createTimeMs,
+                     String executeSql) {
+        super(jobId, jobName, jobStatus, currentDbName, comment, createUser,
```

Comment:
You use `label` as job's name.
But job's name need to be global unique.
So I think job name should be `dbid+label+uuid`